AllReadyApp/Web-App/AllReady/Views/Home/Index.cshtml

```diff
@@ -21,9 +23,61 @@
         <div class="activity-results" data-bind="if: loadingDone">
             <h3>Activites near <span data-bind="text: searchTerm"></span></h3>
             <div class="activity-wrapper" data-bind="foreach: activities">
-                <p><a data-bind="attr: { href: '/Activity/' + Id }, text: Title"></a></p>
+                <p>
+                    <a data-bind="attr: { href: '/Activity/' + Id }, text: Title"></a>
+                </p>
             </div>
             <a data-bind="attr: { href: '/Activity'}">See more activities</a>
         </div>
     </div>
+</div>*@
+
+<div class="col-md-6">

```

Comment:
To keep Bootstrap happy, wrap this in a `<div class="row">`. Also, change the class to `col-md-12` since each row in the Bootstrap grid system should specify 12 columns (and there's no good reason not to here).